### PR TITLE
[CFM-1344] Adjust angular widgets for accessibility.

### PR DESCRIPTION
**Pull request by anvmn**
https://issuetracker.amplexor.com/jira/browse/CFM-1344

### CODE REVIEW

File: project/profiles/capacity4more/modules/c4m/vocab/c4m_vocab_widget/templates/c4m_vocab_widget.tpl.php [hunk 1]
```diff
@@ -5,7 +5,7 @@
  * Template to render the Vocabulary widget.
  */
 ?>
-<div
+<div class="c4m_vocab_description"
```
Comment (ordavidil) [inline project/profiles/capacity4more/modules/c4m/vocab/c4m_vocab_widget/templates/c4m_vocab_widget.tpl.php]: html classes convention is to use a regular dash `-`

File: project/profiles/capacity4more/themes/c4m/kapablo/build/src/stylesheets/states/_activity_global.scss [hunk 2]
```diff
@@ -75,10 +75,26 @@ body {
...
+    .c4m_vocab_language,
+    .c4m_vocab_geo,
+    .c4m_vocab_description,
     .pane-quick-form,
     .field-name-activity-stream-by-topic,
     .pane-content .activity-stream {
       display: none;
     }
+
+    .btn-group {
```
Comment (ordavidil) [inline project/profiles/capacity4more/themes/c4m/kapablo/build/src/stylesheets/states/_activity_global.scss]: This rule looks too general

File: project/profiles/capacity4more/themes/c4m/kapablo/build/src/stylesheets/states/_activity_global.scss [hunk 3]
```diff
@@ -75,10 +75,26 @@ body {
...
+      &.categories {
+        display: none;
+      }
+    }
+
+    .input-wrapper {
```
Comment (ordavidil) [inline project/profiles/capacity4more/themes/c4m/kapablo/build/src/stylesheets/states/_activity_global.scss]: This rule looks too general

**Comment by anvmn** [comment]
selection_148

**Comment by anvmn** [comment]
selection_149

**Comment by anvmn** [comment]
selection_150

**Comment by anvmn** [comment]
selection_151

**Comment by anvmn** [comment]
Tags widget is implemented using 'Select2' - https://github.com/select2/select2. Version 3.4.8.
selection_153


Don't really see how can I effect it's accessibility problems.
Maybe upgrading to higher version (current is 4.0.3), will solve the issues?

**Comment by ordavidil** [comment]
leave the select2 for now.

**Comment by anvmn** [comment]
Last issue with accessibility, is missing legend at OG vocab widget:
selection_154

No title is passed to the template, and as a result, legend is not generated.

**Comment by ordavidil** [comment]
@anvmn 
does all the elements has the `title` attribute ?

**Comment by anvmn** [comment]
@ordavidil 
Short answer is no.

File: project/profiles/capacity4more/modules/c4m/widgets/c4m_document_widget/templates/c4m_document_widget.tpl.php [hunk 4]
```diff
@@ -35,7 +35,8 @@
       </script>
       <a href="JavaScript://" onclick="Drupal.media.popups.mediaBrowser(c4m_attachment_overlay,media_browser_options)"
          ng-click="setFieldName('<?php print $field_name; ?>')" id="link-<?php print $field_name; ?>"
-         ng-class="{'active-library-link':fieldName === '<?php print $field_name; ?>'}">
+         ng-class="{'active-library-link':fieldName === '<?php print $field_name; ?>'}"
+      title="<?php print t('Select a document from the library') ?>">
```
Comment (ordavidil) [inline project/profiles/capacity4more/modules/c4m/widgets/c4m_document_widget/templates/c4m_document_widget.tpl.php]: can you please notice the indentation?

File: project/profiles/capacity4more/modules/c4m/og/c4m_og_vocab/templates/c4m_categories_vocab_widget.tpl.php [hunk 5]
```diff
@@ -59,19 +64,20 @@ class="form-control" type="text" placeholder="Search" title="Search"/>
...
             <ul ng-show="item.selected == true" class="indent">
               <li ng-repeat="child in item.children">
-                <label>
+                <label title="{{child.label}}">
                   <input type="checkbox" data-target="{{item.id}}"
                          ng-name="type"
                          ng-model="model.<?php print $vocabulary_machine_name; ?>[child.id]"
                          ng-change="updateSelectedTerms(child.id, '<?php print $vocabulary_machine_name; ?>')"
-                         title="{{child.label}}">
+                         title="Check">
```
Comment (ordavidil) [inline project/profiles/capacity4more/modules/c4m/og/c4m_og_vocab/templates/c4m_categories_vocab_widget.tpl.php]: => `Check {{child.label}}` 
looks better

**Pull request merged**